cli/.js/src/main/scala/aqua/Utils.scala

```diff
@@ -1,8 +1,13 @@
 package aqua
 
-import scribe.Level
+import aqua.js.{AvmLogLevel, FluenceJSLogLevel, Meta, Module}
+import fs2.io.file.Path
+import scribe.{Level, Logging}
+
+import scala.util.Try
+
+object Utils extends Logging {
```

Comment:
For me, any `Utils`-named class/object seems like a tech debt. It addresses what problem? How does it help?